vembrane/__init__.py

```diff
@@ -401,13 +401,22 @@ def main():
             ),
         )
 
+        records = filter_vcf(
+            vcf,
+            args.expression,
+            args.annotation_key,
+            keep_unmatched=args.keep_unmatched,
+        )
+
+        try:
+            first_record = next(records)
```

Comment:
```suggestion
            first_record = list(islice(records, 1))
```
(and `from itertools import islice`)

Reply:
Otherwise the `chain` below will not work and we `exit(1)` for empty VCFs, which are valid input.